docs/readme_login.md

```diff
@@ -0,0 +1,304 @@
+            EOSManager.Instance.AuthLinkExternalAccountWithContinuanceToken(loginCallbackInfo.ContinuanceToken, 
+                                                                            LinkAccountFlags.NoFlags,
+                                                                            (Epic.OnlineServices.Auth.LinkAccountCallbackInfo linkAccountCallbackInfo) =>
+            {
+                
+                if (linkAccountCallbackInfo.ResultCode == Result.Success)
+                {
+                    // 🔵 Proceed to Connect Login
+                    StartConnectLoginWithLoginCallbackInfo(loginCallbackInfo);
+                }
+                else return; // 🔴 Epic Account wasn't linked
+            });
+        }
+
+        else return; 🔴 Other fail cases of the external login attempt
+    }
+```
+```cs    
+    private void StartConnectLoginWithLoginCallbackInfo(LoginCallbackInfo loginCallbackInfo)
+    {
+        // 🔵 Attempts Connect Login
+        EOSManager.Instance.StartConnectLoginWithEpicAccount(loginCallbackInfo.LocalUserId, (Epic.OnlineServices.Connect.LoginCallbackInfo connectLoginCallbackInfo) =>
+        {
+            if (connectLoginCallbackInfo.ResultCode == Result.Success)
+            {
+                // 🟢 Login flow completed and succeeded
+            }
+            
+            // 🟡 There is no user created for this game on this Epic Account yet
+            else if (connectLoginCallbackInfo.ResultCode == Result.InvalidUser)
+            {
+                // 🟡 Ask user if they want to connect; sample assumes they do and creates an user
+                EOSManager.Instance.CreateConnectUserWithContinuanceToken(connectLoginCallbackInfo.ContinuanceToken, (Epic.OnlineServices.Connect.CreateUserCallbackInfo createUserCallbackInfo) =>
+                {
+                    // 🔵 Attempts Connect Login again after user created for the account
+                    EOSManager.Instance.StartConnectLoginWithEpicAccount(loginCallbackInfo.LocalUserId, (Epic.OnlineServices.Connect.LoginCallbackInfo retryConnectLoginCallbackInfo) =>
+                    {
+                        if (retryConnectLoginCallbackInfo.ResultCode == Result.Success)
+                        {
+                            // 🟢 Login flow completed and succeeded
+                        }
+                        
+                        else return; // 🔴 Connect login remain failing
+                    
+                    });
+                });
+            }
+        });
+    }
+```      
+❗*What happens after login should be tailored to fit the game's flows, such as configure UI, open main scene, etc*   
+*For some fail cases we pass an empty callbackinfo `LoginCallbackInfo blankCallbackInfo = new LoginCallbackInfo()` into callbacks to get better debug visibilty*    
+*They are omitted here to keep the file concise, more details could be found in plugin scripts* 
```

Comment:
I think we could add some references to the exact script that has the details to aid the user in finding it.